identity/backend/src/main/java/io/camunda/identity/usermanagement/repository/MembershipRepository.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.usermanagement.repository;
+
+import io.camunda.identity.usermanagement.Group;
+import java.util.List;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class MembershipRepository {
+  public static final String DEF_USERS_GROUPS_QUERY =
+      "select g.id, g.group_name"
+          + " from groups g, group_members gm"
+          + " where gm.username = ? and g.id = gm.group_id";
+
+  private final JdbcTemplate jdbcTemplate;
```

Comment:
I prefer to keep it as it is in this PR because maybe with next task (customised fields for User) I change these repository classes to use spring data jpa instead of  jdbc.